scss/_navbar.scss

```diff
@@ -178,13 +223,12 @@
 }
 
 
-// Navbar text
+// Navbar text and item
```

Comment:
`// Navbar text` comes from Bootstrap. Rather add `// Navbar item` as a Boosted mod after this section.